guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -184,4 +178,13 @@ endif::[]
 # {foreman-installer}
 ----
 endif::[]
+ifdef::foreman-el,katello,satellite[]
+. On {EL} 8: Switch to the PostgreSQL 13 module:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf -y module switch-to postgresql:13
+----
+. If you are using an external database, upgrade your database to PostgreSQL 13.
```

Comment:
This needs to happen before "Run the installer" for Foreman/Katello, and before "maintain check" for Satellite, otherwise the upgrade will fail.